src/pages/get-eth.js

```diff
@@ -244,160 +254,215 @@ const GetETHPage = ({ data }) => {
+          <h3>
+            <Translation id="page-get-eth-traditional-currencies" />
+          </h3>
+          <p>
+            <Translation id="page-get-eth-traditional-payments" />
+          </p>
           <CardList content={decentralizedExchanges} />
-          <h3>Buy with other crypto</h3>
-          <p>Swap your tokens for other people's ETH. And vice versa.</p>
+          <h3>
+            <Translation id="page-get-eth-other-cryptos" />
+          </h3>
+          <p>
+            <Translation id="page-get-eth-swapping" />
+          </p>
           <CardList content={tokenSwaps} />
           <Warning>
-            These DEXs aren't for beginners as you'll need some ETH to use them.
+            <Translation id="page-get-eth-warning" />
           </Warning>
         </RightColumn>
       </TwoColumnContent>
       <Divider />
       <Content>
-        <h2>Keeping your ETH safe</h2>
+        <h2>
+          <Translation id="page-get-eth-keep-it-safe" />
+        </h2>
       </Content>
       <TwoColumnContent>
         <WalletLeftColumn>
           <WalletImage fluid={data.wallet.childImageSharp.fluid} />
-          <h3>Community posts on security</h3>
+          <h3>
+            <Translation id="page-get-eth-community-safety" />
+          </h3>
           <CardList content={safetyArticles} />
         </WalletLeftColumn>
         <RightColumn>
           <p>
-            Ethereum and ETH aren't controlled by any government or company -
-            they are decentralized. This means ETH's open to everyone to use.
+            <Translation id="page-get-eth-description" />
           </p>
           <p>
-            But this also means you need to take the security of your funds
-            seriously. With ETH, you’re not trusting a bank to look after your
-            money, you’re trusting yourself.
+            <Translation id="page-get-eth-security" />
           </p>
-          <h3>Protect your ETH in a wallet</h3>
+          <h3>
+            <Translation id="page-get-eth-protect-eth-in-wallet" />
+          </h3>
           <p>
-            If you plan on buying a lot of ETH you may want to keep it in a
-            wallet you control, not an exchange. That's beacuse an exchange is a
-            likely target for hackers. If a hacker gains access, you could lose
-            your funds. Alternatively, only you have control of your wallet.
+            <Translation id="page-get-eth-protect-eth-desc" />
           </p>
-          <h3>Your ETH address</h3>
+          <h3>
+            <Translation id="page-get-eth-your-address" />
+          </h3>
           <p>
-            When you download a <Link to="/wallets/">wallet</Link>, it will
-            create a public ETH address for you. Here's what one looks like:
+            <Translation id="page-get-eth-your-address-desc" />{" "}
+            <Link to="/wallets/">
+              <Translation id="wallet" />
```

Comment:
This is causing a console error in the [preview deploy](https://deploy-preview-1420--ethereumorg.netlify.app/en/get-eth/):
```
VM53 react_devtools_backend.js:2273 No key "wallet" in en.json. Cannot provide a default message.
```
Make sure you're checking your work in the preview deploys.

Reply:
(It's because there's no `wallet` id in en.json)